cli/rbd/mirror/bootstrap.py

```diff
@@ -0,0 +1,53 @@
+from copy import deepcopy
+
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+
+
+class Bootstrap(Cli):
+    """This module provides CLI interface to manage rbd pool peer bootstrap commands."""
+
+    def __init__(self, nodes, base_cmd):
+        super(Bootstrap, self).__init__(nodes)
+        self.base_cmd = base_cmd + " bootstrap"
+
+    def create(self, **kw):
+        """Wrapper for pool peer bootstrap create.
+        Create a peer bootstrap token to import in a remote cluster.
+        Args:
+            kw: Key value pair of method arguments
+        Example::
+        Supported keys:
+            pool-name: Name of the pool of which peer is to be bootstrapped.
+            site-name: Name of the site for rbd-mirror
+                       name of both sites should be the same
+            token-path: path to store bootstrap token
+            See rbd help mirror pool peer bootstrap create for more supported keys
+        """
+        kw_copy = deepcopy(kw)
+        pool_name = kw_copy.pop("pool-name", "")
+        token_path = kw_copy.pop("token-path", "")
+        cmd = f"{self.base_cmd} create {pool_name} {build_cmd_from_args(**kw_copy)} > {token_path}"
```

Comment:
redirecting output is not required here. This will limit the functionality of this method. Instead collect output and create file.

Reply:
Changes done.